operator/main.py

```diff
@@ -837,6 +837,26 @@ def delete_storage_class(hostvol_name, _):
         volname=hostvol_name
     ))
 
+def csi_driver_object_api_version():
+    """
+    Return API Version of CSI Driver object"
+    """
+
+    cmd = ["kubectl", "get", "csidriver", "-nkadalu", "-ojson"]
```

Comment:
- incorrect command, csidriver is a cluster scoped resource and more over search for kadalu
- if it's a typo just change `-nkadalu` => `kadalu`

Reply:
Ack.